integrations/langchain-js/README.md

```diff
@@ -0,0 +1,76 @@
+# @braintrust/langchain-js
+
+SDK for integrating [Braintrust](https://braintrust.dev) with [LangChain.js](https://langchain.com/js). This package provides a callback handler to automatically log LangChain.js executions to Braintrust.
+
+## Installation
+
+```bash
+npm install @braintrust/langchain-js
+# or
+yarn add @braintrust/langchain-js
+# or
+pnpm add @braintrust/langchain-js
+```
+
+## Requirements
+
+- Node.js >= 16
+- LangChain.js >= 0.3.0
+
+## Usage
+
+First, make sure you have your Braintrust API key set in your environment:
+
+```bash
+export BRAINTRUST_API_KEY="your-api-key"
+```
+
+### Basic Usage
+
+```typescript
+import { ChatOpenAI } from "@langchain/openai";
+import { BraintrustCallbackHandler } from "@braintrust/langchain-js";
+
+// Create the callback handler (optionally pass in a custom logger)
+const handler = new BraintrustCallbackHandler();
+
+// Initialize your LangChain components with the handler
+const model = new ChatOpenAI({
+  callbacks: [handler],
+});
+
+// Use LangChain as normal - all calls will be logged to Braintrust
+const response = await model.invoke("Tell me a joke about bears", {
+  callbacks: [handler],
```

Comment:
I did attempt to get a global handler setup. TL;DR:

- js version doesn't have a `register_configure_hook` like modern langchain py https://github.com/langchain-ai/langchain/blob/master/libs/community/langchain_community/callbacks/manager.py#L35
- others have mixed results like wandb prefer instructing users to include the callback in the calls
- openinference does patch https://github.com/Arize-ai/openinference/blob/main/js/packages/openinference-instrumentation-langchain/src/instrumentationUtils.ts#L18

I would prefer to avoid monkey patching, and instead I suspect we'll want to contribute by adding a `register_configure_hook` or similar in the js version.